python/pyxir/contrib/target/components/DPUCAHX8H/u50lv_op_support.py

```diff
@@ -0,0 +1,235 @@
+def pooling_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """Check whether we can execute the provided Pooling operator
+    on the DPUCAHX8H-u50lv target"""
+    return is_pooling_supported(
+        X=X,
+        bXs=bXs,
+        tXs=tXs,
+        channel_parallel=16,
+        bank_depth=2048,
+        max_pool_min_kernel=1,
+        max_pool_max_kernel=8,
+        max_pool_min_stride=1,
+        max_pool_max_stride=8,
+        avg_pool_min_kernel=2,
+        avg_pool_max_kernel=8,
+        avg_pool_min_stride=1,
+        avg_pool_max_stride=8,
+    )
+
+
+@pyxir.register_op_support_check("DPUCAHX8H-u50lv", "Mean")
+def mean_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """Check whether we can execute the provided Mean operator
+    on the DPUCAHX8H-u50lv target"""
+    axes = X.attrs["axes"]
+    keepdims = X.attrs["keepdims"]
+    return len(axes) == 2 and keepdims
+
+
+@pyxir.register_op_support_check("DPUCAHX8H-u50lv", "LeakyReLU")
+def leaky_relu_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """Check whether we can execute the provided LeakyRelu operator
+    on the DPUCAHX8H-u50lv target"""
+    assert len(bXs) == 1
+    bX = bXs[0]
+    alpha = X.attrs["alpha"]
+    # LeakyRelu not supported after depthwise conv2d
+    return math.isclose(alpha, 0.1, rel_tol=1e-5) and bX.type[0] in [
+        "Convolution",
+        "Conv2DTranspose",
+        "BatchNorm",
+        "BiasAdd",
+        "Scale",
+    ]
+
+
+@pyxir.register_op_support_check("DPUCAHX8H-u50lv", "pReLU")
+def prelu_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """Check whether we can execute the provided pRelu operator
+    on the DPUCAHX8H-u50lv target"""
+    # Only LeakyReLU: alpha == 0.1 supported
+    assert len(bXs) == 1
+    bX = bXs[0]
+    alpha = X.attrs["alpha"]
+    # LeakyRelu not supported after depthwise conv2d
+    return math.isclose(alpha, 0.1, rel_tol=1e-5) and bX.type[0] in [
+        "Convolution",
+        "Conv2DTranspose",
+        "BatchNorm",
+        "BiasAdd",
+        "Scale",
+    ]
+
+
+@pyxir.register_op_support_check("DPUCAHX8H-u50lv", "ReLU")
+def relu_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """Check whether we can execute the provided ReLU operator
+    on the DPUCAHX8H-u50lv target"""
+    assert len(bXs) == 1
+    bX = bXs[0]
+    return bX.type[0] in set(
+        ["Convolution", "Conv2DTranspose", "Eltwise", "BatchNorm", "BiasAdd", "Scale"]
+    )
+
+
+@pyxir.register_op_support_check("DPUCAHX8H-u50lv", "ReLU6")
+def relu6_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """Check whether we can execute the provided ReLU6 operator
+    on the DPUCAHX8H-u50lv target"""
+    assert len(bXs) == 1
+    bX = bXs[0]
+    return bX.type[0] in set(
+        ["Convolution", "Conv2DTranspose", "Eltwise", "BatchNorm", "Scale"]
```

Comment:
Relu6 is not supported after eltwise according to the doc. You can check with a unit test. We will also see this issue in the operator support of other DPUs.